app/utils/behandlinger.utils.ts

```diff
@@ -33,6 +34,20 @@ export function overlapper(
   return true;
 }
 
+export function erInnenforBehandlingsperiode(
```

Comment:
smør på flesk men nice to have? 
```suggestion
export function erMeldekortInnenforBehandlingsperiode(
```